packages/fx-core/src/plugins/resource/bot/index.ts

```diff
@@ -207,54 +206,19 @@ export class TeamsBot implements Plugin {
       return await runWithExceptionCatching(
         context,
         async () => {
-          if (isCLIDotNetEnabled()) {
-            const node = new QTreeNode({ type: "group" });
-            node.condition = showNotificationTriggerCondition;
-
-            const dotnetNode = new QTreeNode(createHostTypeTriggerQuestionForVS());
-            dotnetNode.condition = {
-              validFunc: async (input: unknown, inputs?: Inputs) => {
-                if (inputs && inputs[CoreQuestionNames.Runtime] === "dotnet") {
-                  return undefined;
-                } else {
-                  return "Runtime is not .net.";
-                }
-              },
-            };
-            node.addChild(dotnetNode);
-
-            const nodejsNode = new QTreeNode(
-              createHostTypeTriggerQuestion(context.answers?.platform)
+          const res = new QTreeNode({
+            type: "group",
+          });
+          const runtimes = [Runtime.Dotnet, Runtime.Node];
```

Comment:
Nit: Consider using the Enum type itself to reflect this runtime list if feasible.

Reply:
Some e2e tests fail at scaffolding stage after I switched to looping using Enum and I think there exists bugs in either question model or cli after doing some experiments. Using array can be a workaround here. May need discussions offline.